src/settings/utils/time.js

```diff
@@ -1,9 +1,28 @@
 import moment from 'moment';
 
+import {
+  localizer,
+  MONDAY,
+  TUESDAY,
+  WEDNESDAY,
+  THURSDAY,
+  FRIDAY,
+  SATURDAY,
+  SUNDAY,
+} from '../constants';
+
 // eslint-disable-next-line import/prefer-default-export
 export const setTimezoneOffset = (date) => {
   const momentDate = moment(date);
   const utcOffset = -(momentDate.utcOffset() / 60);
 
   return momentDate.add(utcOffset, 'hours');
 };
+
+export const getWeekdays = (locale) => {
+  const weekdays = [MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY];
+  const method = (!locale || localizer.startOfWeek(locale) === 0) ? 'unshift' : 'push';
+  weekdays[method](SUNDAY);
```

Comment:
Could you please add empty line for separate business logic?

Reply:
Done